src/services/enum-utilities.ts

```diff
@@ -127,6 +127,11 @@ export default class EnumUtilities {
     return (item.name === FilingTypes.CORRECTION)
   }
 
+  /** Returns True if filing is a Dissolution. */
+  static isTypeDissolution (item: any): boolean {
```

Comment:
I don't see this being used anywhere Sev! Maybe we'll be using it later?

Reply:
Yeah. I was in this code already for amalgamations and I noticed this function was missing 🤷‍♂️